doc/interop-test-descriptions.md

```diff
@@ -1143,10 +1147,14 @@ Procedures:
       }
     }
     ```
-The call carries a reference to receive the load report, e.g. using CallOptions.
-The reference will be passed to the custom LB policy as part of the `OrcaOobReportListener` API.
-* Client asserts that, after 1.5 second, the latest OOB load report received is equal to the test load report.
-* Client sends another unary call to the server. The call request sets `orca_oob_report` to a 
+The test then blocks until the first response is received. This means the server lock is acquired.
+It is important that if the stream is completed or terminated with error at this time, 
+the test aborts and fails. 
+2. Client verifies (timeout=5s) that the latest OOB load report received is equal to the test load report.
+To do verification, client may inject a callback to the custom LB policy, or poll the result by
+doing empty unary calls that carries a reference, e.g. using CallOptions, that will be filled in
+by the custom LB policy as part of the `OrcaOobReportListener` API.
+3. Client then sends another request on the stream to the server. The request sets `orca_oob_report` to a 
```

Comment:
Delete "The request sets `orca_oob_report` to a different test load report." We're not explaining why the test is written the way it is. We're saying what the test has to do. "You do things things, and you have verified the behavior is correct."